dipy/workflows/io.py

```diff
@@ -205,3 +207,39 @@ def run(self, data_names, out_dir=''):
             # the same process, we don't have the env variable pointing to the
             # wrong place
             self.load_module('dipy.data.fetcher')
+
+
+class SplitFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'split'
+
+    def run(self, input_files, grad_dir=0, out_dir='',
```

Comment:
I find `grad_dir` to be confusing, because I'd expect something like a 3D vector pointing in some direction, rather than an integer. How about `vol_idx` instead?

Reply:
Agree, +1

Reply:
+1 we can use the same script to split streamline files too.

Reply:
done!